### PR TITLE
WRAPPER WIDGET Configurable margin in widgets [sc-196869]

### PR DESCRIPTION
**Pull request by atoom75**
Allow to pass `margin` as props for `WrapperWidgetUI`

### CODE REVIEW

**Review by alasarr — APPROVED** [review]

File: packages/react-ui/src/widgets/WrapperWidgetUI.js [hunk 1]
```diff
@@ -43,14 +43,14 @@ const useStyles = makeStyles((theme) => ({
     width: '100%',
     height: theme.spacing(0.25)
   },
-  header: ({ expanded }) => ({
+  header: ({ expanded, margin }) => ({
```
Comment (padawannn) [inline packages/react-ui/src/widgets/WrapperWidgetUI.js:46]: This margin is never used

**Review by padawannn — CHANGES_REQUESTED** [review]
I like the approach but with default values, all widgets are 14px lower in height than the current widgets. I'm wondering if our current widgets are wrong or we need to fix it in this PR

**Comment by Clebal** [comment]
In DrawingToolWidgetUI, we use className prop to override default root styles. It has more possibilities than only margin. What do you think guys? @alasarr @padawannn @atoom75

**Comment by padawannn** [comment]
> In DrawingToolWidgetUI, we use className prop to override default root styles. It has more possibilities than only margin. What do you think guys? @alasarr @padawannn @atoom75

We already had a meeting about this and the idea of this is precisely not to be able to overwrite all the widget styles

**Review by padawannn — APPROVED** [review]
After talking with @alasarr we've decided that we can live with these 14px of height difference

**Pull request merged**